CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # React Native Zcash
 
+## Unreleased
+
+iOS:
+
+- changed: Upgrade ZcashLightClientKit to v2.0.3
+- removed: Remove transaction workaround added previously in v0.6.3
```

Comment:
This isn't correct syntax, and we need to be picky now that we are using automated scripts on changelogs. Do `- changed: (iOS) Upgrade...` or something so the iOS is part of each item, instead of separate.

Reply:
Good idea, fixed